psi4/src/psi4/libmints/twobody.h

```diff
@@ -144,6 +145,12 @@ class PSI_API TwoBodyAOInt {
     /// Significant shell pairs, indexes by shell
     std::vector<std::vector<int>> function_to_function_;
     std::function<bool(int, int, int, int)> sieve_impl_;
+    /// Do Density Screening?
+    bool density_screening_;
+    /// Density Screening Tolerance
+    double density_screening_threshold_;
+    /// Max Density per Shell Pair
+    std::vector<std::vector<double>> max_dens_shell_pair_;
```

Comment:
Move this line of code to be near `function_pair_values_` and its brethren: they're all giving lookup information used by the various screening tests. Similar to those functions, this vector of vectors should be flatten into a simple vector.